### DISCUSSION TITLE
人工智能导论 - 北京工业大学课程资料共享计划

### DISCUSSION THREAD

**Discussion by giscus** [Nagi-ovo/BJUT-Helper#19] [Announcements]
# 人工智能导论 - 北京工业大学课程资料共享计划

https://helper.bjutswift.cn/%E4%BA%BA%E5%B7%A5%E6%99%BA%E8%83%BD%E5%AF%BC%E8%AE%BA/?h=%E5%AF%BC%E8%AE%BA

**Comment by Nagi-ovo** [comment]
建议是和同学讨论做一份标答出来，然后背下答案，当然最好也能理解
